backend/app/schemas.py

```
@@ -124,44 +123,39 @@ class ProductUpdate(BaseModel):
-    order_items: List[OrderItemCreate]  
-  
-class OrderOut(BaseModel):  
-    order_id: int  
-    user_id: int  
-    status: str  
-   
-  
-    class Config:  
-        from_attributes = True  
+class OrderCreate(BaseModel):
+    order_items: List[OrderItemCreate]
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)